Parse/src/main/java/com/parse/ParseOkHttpClient.java

```diff
@@ -146,9 +146,7 @@ public ParseOkHttpClient(int socketOperationTimeout, SSLSessionCache sslSessionC
     // Set Body
     ParseHttpBody parseBody = parseRequest.getBody();
     ParseOkHttpRequestBody okHttpRequestBody = null;
-    if(parseBody instanceof ParseByteArrayHttpBody) {
-      okHttpRequestBody = new ParseOkHttpRequestBody(parseBody);
-    }
+    okHttpRequestBody = new ParseOkHttpRequestBody(parseBody);
```

Comment:
I think this will cause NPE if `parseBody` is `null` when actually executing the request since all the methods on `ParseOkHttpRequestBody use it: https://github.com/ParsePlatform/Parse-SDK-Android/blob/master/Parse/src/main/java/com/parse/ParseOkHttpClient.java#L281